blocks/_helpers/index.js

```diff
@@ -0,0 +1 @@
+export * from './helpers';
```

Comment:
We could probably do with a bit less indirection here 🙂 

Effectively, we have `_helpers` proxied to `_helpers/helpers.js` by way of `_helpers/index.js`. I appreciate re-exporting small, related modules from an index file, but this seems like [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).

I'd propose these alternatives we put the contents of `blocks/_helpers/helpers.js` directly in `blocks/_helpers.js` and remove directory (YAGNI, this is simplest).

Reply:
Yeah, good points :+1: